apps/coscrad-frontend/src/components/resources/utils/upper-case-first-letter.ts

```diff
@@ -0,0 +1,7 @@
+// Share via lib to frontend and backend
```

Comment:
I'm not actually using this but wanted to keep it as a reminder to include in external lib

Reply:
If this file isn't used, remove it.

Reply:
I see you've done this.